### ISSUE TITLE
Preheat simulator before running scan

### ISSUE THREAD

**Issue by Antondomashnev** [fastlane/fastlane#8779]
🔑
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I've recently had an issue while running `scan` on CI with the simulator is failed to start. And I've found the issue https://github.com/fastlane/fastlane/issues/1943 which has the solution, but I wonder if it makes sense to add an optional parameter for `scan` to preheat simulator, to avoid duplication of the script usage. I would love to help with this issue if you think it makes sense.

**Comment by lyndsey-ferguson** [comment]
I'm not sure if this is the same issue, but my tests were failing because `xcodebuild` gave up waiting on the Simulator to start on a slower Mac Mini. I fixed the issue by asking `scan` to start up the simulator before it ran `xcodebuild`:
```
ENV['FASTLANE_EXPLICIT_OPEN_SIMULATOR'] = "true"
```

**Comment by Antondomashnev** [comment]
@lyndsey-ferguson hah, maybe that would work, thanks.
